### DISCUSSION TITLE
Works perfectly on Void Linux with LBP 2900

### DISCUSSION THREAD

**Discussion by unixanarchist** [mounaiban/captdriver#12] [Ideas]
Thank you for this driver and extensive documentation! I have seen the original edition by other people but I couldn't figure out why it wasn't working, the installation instructions were pretty hard to understand. I will contribute to the wiki about Void Linux support because people need to know whether or not their distro is supported. I might also translate this to russian, Canon printers are pretty popular in my country. Thank you again and expect some pull requests soon!

**Comment by mounaiban** [comment]
Glad that the driver works on yet another OS! Also thanks for introducing me to Void Linux and its X Binary Package System.  Void
looks rather unique at first sight... I'm getting 'radical idea' vibes that remind me of Linux distros like NixOS and, to a lesser extent, Fedora Silverblue.

One more thing: please submit pull requests in the latest development branch. At time of writing it's [0.1.4-dev-m5](https://github.com/mounaiban/captdriver/tree/0.1.4-m5-dev). I have not clarified this anywhere else, but I will as soon as I can.
